### PR TITLE
Converted the new Team member page to markdown

### PR DESCRIPTION
**Pull request by efajardo**
https://twiki.opensciencegrid.org/bin/view/SoftwareTeam/NewTeamMember
- [ ] Add migration header to TWiki document

### CODE REVIEW

File: docs/software/new-team-member.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Setup Instructions for New Team Members
+=======================================
+
+<span class="twiki-macro TOC" title="Contents:"></span>
+
+Accounts, Group Memberships, Certificates, Etc.
+-----------------------------------------------
+
+1. **Computing account at FNAL**
+    -   To get this, follow the instructions at <https://fermi.service-now.com/kb_view.do?sysparm_article=KB0010797>
```
Comment (brianhlin) [inline docs/software/new-team-member.md]: Could you remove the <> around all the links?

File: docs/software/new-team-member.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+2. ssh access to a UW CompSci account, including AFS access
+    - Send email to Tim
+3. Read/write access to the UW Subversion repository;
+    - Send email to Matt 
+4. Grid certificate
+    - <https://twiki.opensciencegrid.org/bin/view/Documentation/CertificateGetWeb> 
+5. Access to FermiCloud
+    - <http://fclweb.fnal.gov/>
+6 Jira ticket system
```
Comment (brianhlin) [inline docs/software/new-team-member.md]: Looks like the list got mucked up at 6 and it's missing a `.`

File: docs/software/new-team-member.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+Setup Instructions for New Team Members
+=======================================
+
+<span class="twiki-macro TOC" title="Contents:"></span>
```
Comment (brianhlin) [inline docs/software/new-team-member.md]: This line can be deleted

File: docs/software/new-team-member.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Accounts, Group Memberships, Certificates, Etc.
+-----------------------------------------------
+
+1. **Computing account at FNAL**
```
Comment (brianhlin) [inline docs/software/new-team-member.md]: Not sure why this is bolded, could you remove the `**`?

**Comment by efajardo** [comment]
@brianhlin, I changed your requests. Can I merge now?

File: docs/software/new-team-member.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+Setup Instructions for New Team Members
+=======================================
+<span class="twiki-macro TOC" title="Contents:"></span>
```
Comment (brianhlin) [inline docs/software/new-team-member.md]: Sorry I wasn't clear earlier, the `<span>` line was the one we don't want!

**Comment by efajardo** [comment]
@brianhlin done, can it be merged now?

**Comment by brianhlin** [comment]
Make sure you add the migration header to the original document and update the spreadsheet with today's date

**Pull request merged**